dashboard/db/migrate/20160721221335_remove_unique_constraint_from_followers_user_student_user_index.rb

```diff
@@ -0,0 +1,11 @@
+class RemoveUniqueConstraintFromFollowersUserStudentUserIndex < ActiveRecord::Migration
+  def up
+    remove_index :followers, [:user_id, :student_user_id]
+    add_index :followers, [:user_id, :student_user_id], unique: false
+  end
+
+  def down
+    remove_index :followers, [:user_id, :student_user_id]
+    add_index :followers, [:user_id, :student_user_id], unique: true
```

Comment:
The `down` migration is problematic, as it cannot run (successfully) after data is added that violates the uniqueness constraint. Really, the migration should probably have removed uniqueness violations (perhaps by deleting the (chronologically) oldest sections) before adding the constraint.

No action required or requested at this point.